### DISCUSSION TITLE
Is PMD 6, officially supported for security fixes?

### DISCUSSION THREAD

**Discussion by naveen-vardhi-glean** [pmd/pmd#4900] [General]
Hi Team,

Firstly, congrats on the launch of PMD 7. 

I am creating a [brew version for PMD 6](https://github.com/Homebrew/homebrew-core/pull/167318), as version 7 is incompatible with 6.

Brew has a policy of having versions which are supported. https://docs.brew.sh/Versions

Please let me know if 6 is supported and security updates will be released.

Thanks,
Naveen

**Comment by jsotuyod** [comment]
Development of PMD 6 has completely stopped and no further releases are planned.
We *may* do a new revision release (ie: 6.55.1) ***if*** there is a major security issue (ie: a new CVE raised against it), but that is to be decided when that happens. There are no commitments to keep supporting PMD 6 as focus lies now on PMD 7.
